cmd/gf/internal/cmd/cmd_build.go

```diff
@@ -190,7 +192,6 @@ func (c cBuild) Index(ctx context.Context, in cBuildInput) (out *cBuildOutput, e
 	}
 	// System and arch checks.
 	var (
-		spaceRegex  = regexp.MustCompile(`\s+`)
```

Comment:
It is OK here in CLI command, as it is executed once.

Reply:
but this regex use in a **for**. i this it will be executed  several times.